janusgraph-hbase-parent/janusgraph-hbase-core/src/test/java/org/janusgraph/graphdb/hbase/HBaseGraphTest.java

```diff
@@ -41,4 +46,17 @@ public void testVertexTTLImplicitKey() { }
     @Override @Test @Ignore("HBase does not support retrieving cell TTL by client")
     public void testEdgeTTLImplicitKey() { }
 
+    @Test
+    public void testHBaseTimestampProvider() {
```

Comment:
This test passes even without the updates to GraphDatabaseConfiguration. It looks like it hits the `else` condition (see [Codecov](https://codecov.io/gh/JanusGraph/janusgraph/src/timestamp-provider/janusgraph-core/src/main/java/org/janusgraph/graphdb/configuration/GraphDatabaseConfiguration.java#L1405)) instead of the `if` condition that I think needs to be checked to test the issue/fix.